regtests/bin/matrix.base

```diff
@@ -2063,7 +2084,83 @@
+       echo "$rtst -w work_b -m grdset_b -f -p $mpi -n $npl -t $nth  -o all $ww3 ww3_ufs1.1"                  >> matrix.body
+       echo "./bin/test.comp ww3_ufs1.1 work_a work_b"                                                        >> matrix.body
+    fi
+  fi
+
+ #Test of UFS applications with ww3_multi_esmf and grib2 output
+  if [ "$ufs" = 'y' ] && [ "$esmf" = 'y' ] && [ "$grib" = 'y' ]
+  then
+    echo ' ' >> matrix.body
+    echo "$rtst -w work_a_esmf -m grdset_a -C ESMF -f -p $mpi -n $npl   -o all $ww3 ww3_ufs1.1"               >> matrix.body
+    if [ "$rstrt_b4b" = 'y' ]
+    then
+       echo "mkdir -p ww3_ufs1.1/work_b_esmf"                                                                 >> matrix.body
+       echo "cp ww3_ufs1.1/work_a_esmf/20130401.030000.restart.glo_1deg ww3_ufs1.1/work_b_esmf/restart.glo_1deg">> matrix.body
+       echo "$rtst -w work_b_esmf -m grdset_b -C ESMF -f -p $mpi -n $npl   -o all $ww3 ww3_ufs1.1"            >> matrix.body
+       echo "./bin/test.comp ww3_ufs1.1 work_a_esmf work_b_esmf"                                              >> matrix.body
+    fi
+  fi
+
+ #GFSv16 setup with ww3_multi and grib2 output
+  if [ "$ufs" = 'y' ] && [ "$grib" = 'y' ]
+  then
+    echo ' ' >> matrix.body
+    echo "mkdir -p ww3_ufs1.2/work_a"                                                                         >> matrix.body
+    echo "cp ww3_ufs1.2/input/rmp_src* ww3_ufs1.2/work_a/"                                                    >> matrix.body
+    echo "$rtst -w work_a -m grdset_a -f -p $mpi -n $npl  -t $nth  -o all $ww3 ww3_ufs1.2"                    >> matrix.body
+    if [ "$rstrt_b4b" = 'y' ]
+    then
+       echo "mkdir -p ww3_ufs1.2/work_b"                                                                      >> matrix.body
+       echo "cp ww3_ufs1.2/input/rmp_src* ww3_ufs1.2/work_b/"                                                 >> matrix.body
+       echo "cp ww3_ufs1.2/work_a/20130401.030000.restart.gnh_10m ww3_ufs1.2/work_b/restart.gnh_10m"          >> matrix.body
+       echo "cp ww3_ufs1.2/work_a/20130401.030000.restart.gsh_15m ww3_ufs1.2/work_b/restart.gsh_15m"          >> matrix.body
+       echo "cp ww3_ufs1.2/work_a/20130401.030000.restart.aoc_9km ww3_ufs1.2/work_b/restart.aoc_9km"          >> matrix.body
+       echo "$rtst -w work_b -m grdset_b -f -p $mpi -n $npl  -t $nth -o all $ww3 ww3_ufs1.2"                  >> matrix.body
+       echo "./bin/test.comp ww3_ufs1.2 work_a work_b"                                                        >> matrix.body
+    fi
+  fi
+
+ #GFSv16 setup with ww3_multi_esmf and grib2 output
+  if [ "$ufs" = 'y' ] && [ "$esmf" = 'y' ] && [ "$grib" = 'y' ]
+  then
+    echo ' ' >> matrix.body
+    echo "mkdir -p ww3_ufs1.2/work_a_esmf"                                                                    >> matrix.body
+    echo "cp ww3_ufs1.2/input/rmp_src* ww3_ufs1.2/work_a_esmf/"                                               >> matrix.body
```

Comment:
Should this be added to the run_test script instead of the matrix.base?

Reply:
I followed the same logic for other tests that have dependencies to use mkdir in matrix.base.
I will move ` echo "cp ww3_ufs1.2/input/rmp_src* ww3_ufs1.2/work_a_esmf/"` to run_test. Good suggestion.

Reply:
@JessicaMeixner-NOAA Done, now I am going to check the fields